### PR TITLE
bug: Unit tests failure required @ContextConfiguration annotation

### PR DESCRIPTION
**Pull request by saif-ericsson**
Two unit tests cannot be run individually. This fix will make those work.
### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: src/test/java/com/ericsson/ei/flowtests/TestConfigs.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+import com.mongodb.MongoClient;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+
+import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.mongo.tests.MongodForTestsFactory;
+import lombok.Getter;
+
+public class TestConfigs {
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/TestConfigs.java]: This file does not seem to be necessary. Maybe you should remove it.

File: src/test/java/com/ericsson/ei/flowtests/FlowTest.java [hunk 2]
```diff
@@ -65,6 +65,7 @@
         "missedNotificationDataBaseName: FlowTest-missedNotifications",
         "rabbitmq.exchange.name: FlowTest-exchange",
         "rabbitmq.consumerName: FlowTestConsumer" })
+
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/FlowTest.java]: any reason for this extra row here?

File: src/test/java/com/ericsson/ei/flowtests/FlowTest2.java [hunk 3]
```diff
@@ -45,6 +45,7 @@
         "missedNotificationDataBaseName: FlowTest2-missedNotifications",
         "rabbitmq.exchange.name: FlowTest2-exchange",
         "rabbitmq.consumerName: FlowTest2Consumer" })
+
```
Comment (evasiba-ericsson) [inline src/test/java/com/ericsson/ei/flowtests/FlowTest2.java]: any reason for this extra row here

File: src/test/java/com/ericsson/ei/flowtests/TestConfigs.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+        }
+
+        try {
+            testsFactory = MongodForTestsFactory.with(Version.V3_4_1);
+            mongoClient = testsFactory.newMongo();
+            String port = "" + mongoClient.getAddress().getPort();
+            System.setProperty("spring.data.mongodb.port", port);
+        } catch (Exception e) {
+            LOGGER.error(e.getMessage(), e);
+            e.printStackTrace();
```
Comment (tobiasake) [inline src/test/java/com/ericsson/ei/flowtests/TestConfigs.java]: Isn't stacktrace logged already in error message?,, the "e" argument.

File: src/test/java/com/ericsson/ei/flowtests/TestConfigs.java [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.mongo.tests.MongodForTestsFactory;
+import lombok.Getter;
+
+public class TestConfigs {
+
+    private static AMQPBrokerManager amqpBroker;
+    private static MongodForTestsFactory testsFactory;
+    private static ConnectionFactory cf;
```
Comment (tobiasake) [inline src/test/java/com/ericsson/ei/flowtests/TestConfigs.java]: name variable more readable -> "connectionFactory"

File: src/test/java/com/ericsson/ei/flowtests/TestConfigs.java (hunk 1 above)
Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/flowtests/TestConfigs.java]: removed

File: src/test/java/com/ericsson/ei/flowtests/FlowTest.java (hunk 2 above)
Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/flowtests/FlowTest.java]: removed

File: src/test/java/com/ericsson/ei/flowtests/FlowTest2.java (hunk 3 above)
Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/flowtests/FlowTest2.java]: removed

File: src/test/java/com/ericsson/ei/flowtests/TestConfigs.java (hunk 5 above)
Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/flowtests/TestConfigs.java]: file is removed and the variable is changed in the corresponding file at /utils

File: src/test/java/com/ericsson/ei/flowtests/TestConfigs.java (hunk 4 above)
Comment (saif-ericsson) [reply src/test/java/com/ericsson/ei/flowtests/TestConfigs.java]: file is removed and the variable is changed in the corresponding file at /utils

**Comment by evasiba-ericsson** [comment]
Changes in pull requests were merged by someone else. Therefore we close this.

**Pull request closed without merging**